Templates/Dotnet/ModuleBuilderModule/.template.config/template.json

```
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "identity": "PoshCode.ModuleBuilderModule",
+  "author": "PoshCode",
+  "tags": {
+    "language": "Powershell",
+    "type": "project"
+  },
+  "classifications": ["Powershell", "Modules"],
+  "name": "ModuleBuilder Module Generator",
+  "shortName": "ModuleBuilderModule",
```

Can we use a less verbose shortname here?
```suggestion
  "shortName": "PSMBModule",
```